src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1528,4 +1528,80 @@ private class ACCT_IndividualAccounts_TEST {
         System.assertEquals('Ms. Betty, Lily and Tom White Family', requeryFinalAccount.Name);
 
     }
+
+    /*********************************************************************************************************
+    * @description In a multi-currency enabled org, create a contact with different currency than the default one.
+    * Make sure that the account automatically created and the contact have the same currency.
+    */
+    @isTest
+    private static void insertContactMultiCurrencyEnabled() {
+        if (UserInfo.isMultiCurrencyOrganization()) {
+            //Test data setup
+            String nonDefaultCurrencyCode = '';
+            String qryCurrencyTypeString = 'SELECT IsoCode FROM CurrencyType WHERE IsCorporate = false LIMIT 1';
```

Comment:
More out of curiosity here, is there a reason we are using the database.query and not a [SELECT Isocode From.... type of query here? Nothing I would require changing, just curious.

Reply:
Great question! The reason here is because those currency fields are not available unless an org enables multi-currency. The build will fail if we use [SELECT...FROM... ].

Reply:
aaah yes thank you :)